packages/wallets/src/components/WalletsAddMoreCarousel/WalletsAddMoreCarousel.scss

```diff
@@ -21,7 +21,6 @@
     &__carousel {
         overflow: hidden;
         background-color: #fff;
-        padding: 3.2rem 0;
```

Comment:
Removing this padding will clip the box-shadow. We still need the padding-bottom here.

Reply:
Was referring to the figma design for this, but seems like it's clipped there too. Will update it and raise this up with the designers

<img width="989" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/e46071e7-ef5f-4b32-88f1-e421d0e7acbe">